<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogs::QtAppearanceOptionPage</class>
 <widget class="QWidget" name="Dialogs::QtAppearanceOptionPage">
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="useDefaultLabel">
       <property name="text">
        <string>Use system default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fontLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Font</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="fontHorizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fontPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>More options ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="fontCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="widgetStyleLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Widget style</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="widgetStyleComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="widgetStyleCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="styleSheetLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Style sheet</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Widgets::PathSelection" name="styleSheetPathSelection" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="styleSheetCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="paletteLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Palette</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QToolButton" name="paletteToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>select ...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="paletteCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="iconThemeLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Icon theme</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="iconThemeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="iconThemeCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>These settings take effect after restarting the application.
These settings might be overwritten by your Qt 5 platfrom integration plugin and hence have no effect.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::PathSelection</class>
   <extends>QWidget</extends>
   <header>widgets/pathselection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fontCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontPushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetStyleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetStyleComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paletteCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>paletteToolButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>styleSheetCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>styleSheetPathSelection</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconThemeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconThemeComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
